packages/web-worker/src/webpack-parts/loader.ts

```diff
@@ -64,7 +64,8 @@ export function pitch(
   );
 
   const workerOptions = {
-    filename: addWorkerSubExtension(compiler.options.output!.filename!),
+    filename: addWorkerSubExtension(compiler.options.output!
+      .filename! as string),
```

Comment:
I think you can maybe drop the last `!` if you need the cast (shy do you need it?)